### PR TITLE
Replace print() with logger.info()

### PR DESCRIPTION
**Pull request by tituomin**

### CODE REVIEW

File: munigeo/importer/base.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Importer(object):
     def _import_citadel(self, muni, info):
         muni_slug = slugify(muni.name)
 
-        print("\tImporting from Citadel")
+        self.logger.info("\tImporting from Citadel")
```
Comment (juyrjola) [inline munigeo/importer/base.py]: `\t` probably doesn't work too well in logs

File: munigeo/importer/finland.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def _process_muni(self, syncher, feat):
         m = re.match(r'\(2:([\w\s:-]+),([\w\s:-]+)\)', t)
         name_fi = m.groups()[0]
         name_sv = m.groups()[1]
-        print(name_fi)
+        self.logger.info(name_fi)
```
Comment (juyrjola) [inline munigeo/importer/finland.py]: debug

File: munigeo/importer/finland.py [hunk 3]
```diff
@@ -76,15 +76,15 @@ def _setup_land_area(self):
         fin_bbox = Polygon.from_bbox(FIN_GRID)
         fin_bbox.srid = TM35_SRID
         fin_bbox.transform(4326)
-        print("Loading global land shape")
+        self.logger.info("Loading global land shape")
```
Comment (juyrjola) [inline munigeo/importer/finland.py]: debug

File: munigeo/importer/helsinki.py [hunk 4]
```diff
@@ -193,7 +192,7 @@ def _import_division(self, muni, div, type_obj, syncher, parent_dict, feat):
             val = attr_dict['ocd_id']
             args[div['ocd_id']] = val
             obj.ocd_id = ocd.make_id(**args)
-            print("\t%s" % obj.ocd_id)
+            self.logger.info("\t%s" % obj.ocd_id)
```
Comment (juyrjola) [inline munigeo/importer/helsinki.py]: debug, no `\t`

File: munigeo/importer/sync.py [hunk 5]
```diff
@@ -27,7 +32,7 @@ def finish(self):
         delete_list = []
         for obj_id, obj in self.obj_dict.items():
             if not obj._found:
-                print("Deleting object %s" % obj)
+                logger.info("Deleting object %s" % obj)
```
Comment (juyrjola) [inline munigeo/importer/sync.py]: debug

**Comment by tituomin** [comment]
Done

**Comment by tituomin** [comment]
@juyrjola requested changes made.

**Pull request merged**